### PR TITLE
Fixed korean translation error.

### PR DESCRIPTION
**Pull request by kuckjwi0928**
### CATEGORY
- [x] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Fix korean translation error.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE
63783620-48df9d80-c928-11e9-8be4-838560fbc754
AFTER

### TEST PLAN
Pass

### ADDITIONAL INFORMATION
- [x] Has associated issue: Fixes https://github.com/apache/incubator-superset/issues/8124
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/translations/ko/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -917,4027 +917,3125 @@ msgstr ""
...
-#: superset/assets/src/explore/controls.jsx:2180
 msgid "Whether to display the stroke"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2186
 msgid "Filled"
 msgstr ""
 
-#: superset/assets/src/explore/controls.jsx:2188
 msgid "Whether to
```
Comment (dpgaspar) [inline superset/translations/ko/LC_MESSAGES/messages.po]: Ideally these translations (security) should be included on flask-appbuilder

Comment (kuckjwi0928) [reply superset/translations/ko/LC_MESSAGES/messages.po]: https://github.com/dpgaspar/Flask-AppBuilder/tree/master/flask_appbuilder/translations

You mean this repository??

Comment (dpgaspar) [reply superset/translations/ko/LC_MESSAGES/messages.po]: yes

Comment (kuckjwi0928) [reply superset/translations/ko/LC_MESSAGES/messages.po]: Thanks for answer

I will raise pull request soon

**Pull request merged**